### PR TITLE
UI Improvement: Full-width birdseye (if no PTZ cameras) 

### PR DESCRIPTION
**Pull request by skrashevich**
I don't see any reason why needs limit the size of the Birdseye player window

### CODE REVIEW

**Comment by skrashevich** [comment]
> Because there is already a full screen option and opening the screen to have the full screen taken up is a jarring user experience

By default, Birdseye size is 1280x720. Player will not be larger than it.
In your opinion, 1280x720 is a resolution that can occupy the entire screen in 2023 and cause irritation?

**Comment by skrashevich** [comment]
> Also, that is incorrect for the JSMpegPlayer, the JSMpegPlayer takes up the entire screen regardless of the resolution birdseye is set to

That's not now
[ea31166](https://github.com/blakeblackshear/frigate/pull/6817/commits/ea311661d7d9dbe3c2afe61f03f9c4c1cceffa60)

**Comment by NickM-27** [comment]
The correct solution may be to implement the ability to use the mouse to drag the edge of the birdseye window to make it larger / smaller and to remember that value. 

Otherwise it only works for one usecase or the other

**Comment by NickM-27** [comment]
or a separate button similar to how YouTube has a small and large player options

**Comment by skrashevich** [comment]
> or a separate button similar to how YouTube has a small and large player options

Good idea

File: web/src/routes/Birdseye.jsx [hunk 1]
```diff
@@ -80,23 +83,30 @@ export default function Birdseye() {
         {config.birdseye.restream && (
           <select
             className="basis-1/8 cursor-pointer rounded dark:bg-slate-800"
-            value={viewSource}
-            onChange={(e) => setViewSource(e.target.value)}
+            value={isMaxWidth ? "true" : "false"}
+            onChange={(e) => setIsMaxWidth(e.target.value)}
```
Comment (NickM-27) [inline web/src/routes/Birdseye.jsx:87]: This seems like it should not have changed, now there is no way to set the source of the live view?

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by skrashevich** [comment]
frigate-birdseye

**Review by NickM-27 — CHANGES_REQUESTED** [review]
Looks good! I think these comments can be removed

**Review by NickM-27 — APPROVED** [review]

**Comment by skrashevich** [comment]
More pretty
image

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**